components/TimeForm/index.js

```diff
@@ -0,0 +1,42 @@
+
+import styled from "styled-components";
+
+
+
+const FormContainer = styled.fieldset`
+display: flex;
+flex-direction: column;
+`;
+const Label = styled.label`
+  font-weight: bold;
+  height:10px;
+  margin-bottom:8px;
+`;
+const Input = styled.input`
+  height:30px;
+  font-size: inherit;
+  border: 2px solid black;
+  border-radius: 0.5rem;
```

Comment:
If these are the same styles as in the `CustomerForm`, you can reuse the styled components from there.